support/run-tests.sh

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env bash
 # Test the current package under a different kernel.
-# Requires qemu-system-x86_64 and bluebox to be installed.
+# Requires qemu-system-$QEMU_ARCH and bluebox to be installed.
 
 set -eu
 set -o pipefail
 
+qemu_arch="${QEMU_ARCH:-x86_64}"
```

Comment:
Alternatively we could make this a positional argument to `run-tests.sh`. No strong preference either way.

Reply:
Env variable seems good to me.